server/controllers/image.js

```diff
@@ -10,7 +10,22 @@ exports.uploadImage = asyncHandler(async (req, res) => {
   const result = await uploadFile(file);
   res.status(200).json({
     success: {
-      imagePath: result.Location,
+      image: result.Key,
     },
   });
 });
+
+exports.downloadImage = asyncHandler(async (req, res) => {
+  console.log(req.params)
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!

Reply:
Sorry, my bad, going forward I'll be making sure all console logs are cleaned up.